app/views/hyrax/file_sets/_actions.html.erb

```diff
@@ -1,57 +1,6 @@
-<% if can?(:edit, file_set.id) || can?(:destroy, file_set.id) || (Flipflop.enabled?(:annotation) && can?(:download, file_set.id)) %>
-  <div class="btn-group">
-    <button class="btn btn-default dropdown-toggle" data-toggle="dropdown" type="button" id="dropdownMenu_<%= file_set.id %>" aria-haspopup="true" aria-expanded="false">
-      <span class="sr-only"><%= t('.press_to') %> </span>
-      <%= t('.header') %>
-      <span class="caret" aria-hidden="true"></span>
-    </button>
-
-    <ul role="menu" class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenu_<%= file_set.id %>">
-      <% if can?(:edit, file_set.id) %>
-        <li role="menuitem" tabindex="-1">
-          <%= link_to t('.edit'), edit_polymorphic_path([main_app, file_set]),
-            { title: t('.edit_title', file_set: file_set), id: "file_edit" } %>
-        </li>
-
-        <li role="menuitem" tabindex="-1">
-          <%= link_to t('.versions'),  edit_polymorphic_path([main_app, file_set], anchor: 'versioning_display'),
-            { title: t('.versions_title'), id: "file_versions" } %>
-        </li>
-      <% end %>
-
-      <% if can?(:destroy, file_set.id) %>
-        <li role="menuitem" tabindex="-1">
-          <%= link_to t('.delete'), polymorphic_path([main_app, file_set]),
-            method: :delete, 
-            title: t('.delete_title', file_set: file_set),
-            id: "file_destroy",
-            data: { confirm: t('.delete_confirm', file_set: file_set, application_name: application_name) } %>
-        </li>
-      <% end %>
-      
-      <% if can?(:download, file_set.id) %>
-        <li role="menuitem" tabindex="-1">
-          <%= link_to t('.download'),
-                      hyrax.download_path(file_set),
-                      title: t('.download_title', file_set: file_set),
-                      target: "_blank",
-                      id: "file_download",
-                      data: { label: file_set.id } %>
-        </li>
-
-        <% if file_set.pdf? %>
-          <li role="menuitem" tabindex="-1">
-            <%= link_to t('.read_and_annotate'), hyku_addons.pdf_viewer_path(hyrax.download_url(file_set.id).split('/')&.last&.split('?')&.first),  { title: 'Annotate', id: "file_read", target: "_blank" } %>
-          </li>
-        <% end %>
-      <% end %>
-    </ul>
-  </div>
-<% elsif can?(:download, file_set.id) %>
-  <%= link_to t('.download'),
-    hyrax.download_path(file_set),
-    title: t('.download_title', file_set: file_set),
-    target: "_blank",
-    id: "file_download",
-    data: { label: file_set.id } %>
+<% view_condition = [can?(:edit, file_set.id), can?(:destroy, file_set.id), can?(:download, file_set.id), (Flipflop.enabled?(:annotation) && file_set.pdf?)] %>
```

Comment:
Is there somewhere better I can define this variable? I see definitions like this scattered throughout the views, which is not ideal, but maybe necessary.  Let me know if I can improve this.

Reply:
will a helper make sense?